modules/login/templates/form_requestaccount.tpl

```diff
@@ -61,15 +61,19 @@
       <div class="form-group">
           {$form.radiologist.html}
       </div>
+      <div class="form-group">
+        <!-- Google reCaptcha. Don't display if a private key has not bee configured -->
         {if $captcha_key}
-            {* Google reCaptcha *}
-          <div class="form-group">
+{$form.captcha.html}
             <div class="g-recaptcha" data-sitekey="{$captcha_key}"></div>
-            <span id="helpBlock" class="help-block">
-              <b class="text-danger">{$error_message['captcha']}</b>
-            </span>
-          </div>
+            <script src="https://www.google.com/recaptcha/api.js?render={$captcha_key}&onload=onloadCallback nonce={$nonce}"></script>
```

Comment:
This should probably be refactored in a more generic way so that we can use reCAPTCHA in other places if we want to. That may be out of scope for now, plus I'm not really sure the best approach. Open for suggestions.

Reply:
I'm guessing a new react component will be made. 🙂